kylin-spark-project/kylin-spark-common/src/main/java/org/apache/kylin/engine/spark/metadata/cube/PathManager.java

```diff
@@ -49,6 +49,12 @@ public static String getSegmentParquetStoragePath(CubeInstance cube, String segN
         return hdfsWorkDir + "parquet" + File.separator + cube.getName() + File.separator + segName + "_" + identifier;
     }
 
+    public static String getSegmentParquetStoragePath(String hdfsWorkDir, String cubeName, CubeSegment segment) {
+        String segmentName = segment.getName();
+        String identifier = segment.getStorageLocationIdentifier();
+        return hdfsWorkDir + "parquet" + File.separator + cubeName + File.separator + segmentName + "_" + identifier;
```

Comment:
Is there a `File.separator` between `hdfsWorkDir` and `parquet`

Reply:
Actually  the end of `hdfsWorkDir` already comes with a File.separator.

Reply:
OK